watertap/unit_models/electrodialysis_0D.py

```diff
@@ -411,7 +504,18 @@ def build(self):
             units=pyunits.dimensionless,
             doc="water recovery ratio calculated by mass",
         )
-        # TODO: consider adding more performance as needed.
+        self.velocity_diluate = Var(
+            self.flowsheet().time,
+            initialize=0.01,
+            units=pyunits.meter * pyunits.second**-1,
+            doc="Linear velocity of flow",
```

Comment:
```suggestion
            doc="Linear velocity of flow in the diluate",
```

Reply:
Thanks! All these suggestions related to docs have been adopted to 0D.